[2024-04-08T10:53:15.713+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.get_files manual__2024-04-08T10:53:12.317255+00:00 [queued]>
[2024-04-08T10:53:15.789+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.get_files manual__2024-04-08T10:53:12.317255+00:00 [queued]>
[2024-04-08T10:53:15.792+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-04-08T10:53:16.006+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): get_files> on 2024-04-08 10:53:12.317255+00:00
[2024-04-08T10:53:16.060+0000] {standard_task_runner.py:60} INFO - Started process 131929 to run task
[2024-04-08T10:53:16.099+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl', 'get_files', 'manual__2024-04-08T10:53:12.317255+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpg8_crwyn']
[2024-04-08T10:53:16.113+0000] {standard_task_runner.py:88} INFO - Job 19: Subtask get_files
[2024-04-08T10:53:16.354+0000] {task_command.py:423} INFO - Running <TaskInstance: etl.get_files manual__2024-04-08T10:53:12.317255+00:00 [running]> on host 8056db226622
[2024-04-08T10:53:16.728+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='get_files' AIRFLOW_CTX_EXECUTION_DATE='2024-04-08T10:53:12.317255+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-08T10:53:12.317255+00:00'
[2024-04-08T10:53:16.735+0000] {logging_mixin.py:188} INFO - 2 files found in /opt/***/dags/data
[2024-04-08T10:53:16.737+0000] {python.py:202} INFO - Done. Returned value was: ['/opt/***/dags/data/github_events_02.json', '/opt/***/dags/data/github_events_01.json']
[2024-04-08T10:53:16.819+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=etl, task_id=get_files, execution_date=20240408T105312, start_date=20240408T105315, end_date=20240408T105316
[2024-04-08T10:53:16.864+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-08T10:53:16.909+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
